src/current/_includes/v23.1/metric-names.md

```diff
@@ -8,7 +8,10 @@ Name | Description
 `capacity.reserved` | Capacity reserved for snapshots
 `capacity.used` | Used storage capacity
 `capacity` | Total storage capacity
+`changefeed.aggregator_progress` | The earliest timestamp up to which any [aggregator]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}) is guaranteed to have emitted all values for which it is responsible.
```

Comment:
Can you please add a note saying that this metric may regress in case of aggregator / changefeed restarts due to transient error. It might also be worth mentioning that the `changefeed.checkpoint_progress` will not regress and basically tracks the min highwater timestamp for a set of changefeeds. We recommend using `changefeed.checkpoint_progress` over `changefeed.aggregator_progress` for this reason. This came up with doordash so I think it's important to mention here.